### PR TITLE
update dependencies before beta release

### PR DESCRIPTION
**Pull request by kellyselden**

### CODE REVIEW

**Comment by kategengler** [comment]
I looked into this locally. What it says on the tin, "Error: Could not find module `@ember/application/deprecations` imported from `@ember/string/index`", is of course, correct: The included version of `@ember/string` references `@ember/application/deprecations`. The `@ember/string` dependency seems to come from `ember-data`.
So we need a release of `@ember/string` and it looks like that will have to be a major bump because of dropping node versions.
and then we'll need a new `ember-data` beta, updating the dep on `@ember/string`

File: blueprints/app/files/package.json [hunk 1]
```diff
@@ -41,29 +41,29 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.28.0-beta.1",
+    "ember-data": "~4.0.0-beta.0",
```
Comment (kategengler) [inline blueprints/app/files/package.json:44]:
```suggestion
    "ember-data": "~4.0.0-beta.2",
```

File: tests/fixtures/app/defaults/package.json [hunk 2]
```diff
@@ -38,26 +38,26 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.28.0-beta.1",
+    "ember-data": "~4.0.0-beta.0",
```
Comment (kategengler) [inline tests/fixtures/app/defaults/package.json:41]:
```suggestion
    "ember-data": "~4.0.0-beta.2",
```

File: tests/fixtures/app/embroider-no-welcome/package.json [hunk 3]
```diff
@@ -41,28 +41,28 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.28.0-beta.1",
+    "ember-data": "~4.0.0-beta.0",
```
Comment (kategengler) [inline tests/fixtures/app/embroider-no-welcome/package.json:44]:
```suggestion
    "ember-data": "~4.0.0-beta.2",
```

File: tests/fixtures/app/embroider-yarn/package.json [hunk 4]
```diff
@@ -41,29 +41,29 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.28.0-beta.1",
+    "ember-data": "~4.0.0-beta.0",
```
Comment (kategengler) [inline tests/fixtures/app/embroider-yarn/package.json:44]:
```suggestion
    "ember-data": "~4.0.0-beta.2",
```

File: tests/fixtures/app/embroider/package.json [hunk 5]
```diff
@@ -41,29 +41,29 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.28.0-beta.1",
+    "ember-data": "~4.0.0-beta.0",
```
Comment (kategengler) [inline tests/fixtures/app/embroider/package.json:44]:
```suggestion
    "ember-data": "~4.0.0-beta.2",
```

File: tests/fixtures/app/npm/package.json [hunk 6]
```diff
@@ -38,25 +38,25 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.28.0-beta.1",
+    "ember-data": "~4.0.0-beta.0",
```
Comment (kategengler) [inline tests/fixtures/app/npm/package.json:41]:
```suggestion
    "ember-data": "~4.0.0-beta.2",
```

File: tests/fixtures/app/yarn/package.json [hunk 7]
```diff
@@ -38,26 +38,26 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.28.0-beta.1",
+    "ember-data": "~4.0.0-beta.0",
```
Comment (kategengler) [inline tests/fixtures/app/yarn/package.json:41]:
```suggestion
    "ember-data": "~4.0.0-beta.2",
```

**Comment by kategengler** [comment]
Updated ember-data with the updated @ember/string. Still failing under Embroider with ```Uncaught TypeError: Cannot read property 'String' of undefined at webpack://some-cool-app/../node-bDXlVehj.tmp/node_modules/ember-inflector/make-configurable.js?, line 1
```

**Comment by kategengler** [comment]
Looks to be the same errors as in #9597

**Comment by kategengler** [comment]
Updating embroider seems to have made the build failures multiply.

**Comment by kellyselden** [comment]
OK reverting that last commit

**Review by rwjblue — APPROVED** [review]

**Pull request merged**